src/base/bittorrent/torrentcreatorthread.cpp

```diff
@@ -160,3 +160,13 @@ void TorrentCreatorThread::run()
         emit creationFailure(QString::fromStdString(e.what()));
     }
 }
+
+int TorrentCreatorThread::calculatePiecesCount(const QString &inputPath, const int pieceSize)
+{
+    if (inputPath.isEmpty())
+        return 0;
+
+    libt::file_storage fs;
+    libt::add_files(fs, inputPath.toStdString(), fileFilter);
```

Comment:
You misunderstood me, probably... My previous comment was related to affected line only. Here you require toNativePath since you can't be sure what format is used by caller. It's general rule for public methods.